src/js/view/controls/controlbar.js

```diff
@@ -246,6 +269,10 @@ export default class Controlbar {
         _model.change('cues', this.addCues, this);
         _model.change('altText', this.setAltText, this);
         _model.change('customButtons', this.updateButtons, this);
+        _model.change('state', () => {
+            // Check for change of position to counter race condition where state is updated before the current position
+            _model.once('change:position', this.checkDvrLiveEdge, this);
```

Comment:
Inshort we only check for atLiveEdge after the first position update after a state change. We could even remove the listener for onSeeked because state changes occur after seeking but i was worries there might be an edge case where this doesn't hold